### DISCUSSION TITLE
Umbrel migration to Raspiblitz, LND not started

### DISCUSSION THREAD

**Discussion by DuckG** [raspiblitz/raspiblitz#4447] [Q&A] [answered]
Hello,

I did the update from umbrel to raspiblitz. (raspiblitz-v1.8.0c-2022-11-01.img)
the migration was successful, after this migration the LND is not started. Bitcoin v23.0 is up/started. Sync= ok
The umbrel version = lightning v0.17.4-beta.

(Current) LND raspiblitz version =LND 0.15.4-beta

bitcoin v23.0.0 mainnet 12 peers
    /,'        Blocks 832571/832571 Sync OK 100.00%
   /'
               LND 0.15.4-beta
               Not Started | Not Ready Yet | No Data

debug.log;
2024-02-29 18:47:06.432 [INF] CHDB: Checking for schema update: latest_version=29, db_version=31
2024-02-29 18:47:06.432 [ERR] CHDB: Refusing to revert from db_version=31 to lower version=29
2024-02-29 18:47:06.438 [ERR] LTND: unable to open graph DB: channel db cannot revert to prior version
2024-02-29 18:47:06.439 [ERR] LTND: Shutting down because error in main method: unable to open databases: unable to open graph
 DB: channel db cannot revert to prior version
2024-02-29 18:47:06.441 [INF] LTND: Shutdown complete

do i have a manually update of ?[menu, update]
-Release
-Patch
-Bitcoin
-LND
-TOR

In which order to update or which action could maybe solve this problem to help me with the LND-process.? 
LND, then bitcoin, then Release, Patch?
Can anybody help me?

**Comment by rootzoll** [comment]
You used `raspiblitz-v1.8.0c-2022-11-01.img` which has a too old version of LND to run. Please download the v1.10.0 of raspiblitz, flash it to your sd card and run thr recovery/update.

**Comment by DuckG** [comment]
Thanks for your answer,  now i am already started to do a restore of my node. Because i was desperate.
If my restore works include channels recovery then i would like to migrate again to raspiblitz.
How can i do a perfect migration?

fyi the history:
![migration-warning](https://github.com/raspiblitz/raspiblitz/assets/37292164/7cb82625-e503-415d-833d-d6de14e62d21)

First i tried to use the image raspiblitz-fat-v1.10.0-2023-09-22.img.gz, then the migration warning was "raspiblitz might run an Too OLD of an lightning version to migrate your nodes channel data automatically . (see attachment)
1. stop migration.
2. ignore, try anyway.
After this , i stopped and then i used the older image raspiblitz-v1.8.0c-2022-11-01.img.gz.

**Reply by rootzoll** [reply, marked as answer]
Yes this check & warning was build in after 1.8.0 to prevent cases like your. Normally the solution is to wait for the next update of RaspiBlitz that can handle the newer LND version.

**Comment by DuckG** [comment]
Solved by using this version raspiblitz-fat-v1.11.0-2024-04-16.img.
thanks. close.
